tests/util.py

```diff
@@ -322,3 +322,18 @@ def approve_and_execute(transaction_address: str) -> None:
         )
 
     return approve_and_execute
+
+
+def wait_for_slots(slots: int) -> None:
+    import time
+
+    """
+    Blocks waiting until `slots` slots have passed.
+    """
+    slots_beginning = int(solana('get-slot'))
+    while True:
+        # Wait 1 second to poll next slot height (around 2 slots)
+        time.sleep(1)
```

Comment:
Hey, I finally wrote [a blog post](https://ruudvanasseldonk.com/2022/03/20/please-put-units-in-names) about this :grimacing: In this case it says "1 second" in the comment right next to it so I suppose that works as well, I don’t have a strong opinion in this case.